core/services/relay/relay.go

```diff
@@ -112,8 +112,6 @@ var _ loop.Relayer = (*relayerAdapter)(nil)
 // relayerAdapter adapts a [types.Relayer] and [RelayerExt] to implement [loop.Relayer].
```

Comment:
Can we update this comment with the relevant info the disguinishing it from relayServerAdapter?

Reply:
Yep, will do!